fabric-object-builder-api-v1/src/main/resources/fabric-object-builder-api-v1.accesswidener

```diff
@@ -1,5 +1,6 @@
 accessWidener	v1	named
 extendable  method    net/minecraft/block/AbstractBlock$Settings    <init>  (Lnet/minecraft/block/Material;Ljava/util/function/Function;)V
 extendable  method    net/minecraft/block/AbstractBlock$Settings    <init>  (Lnet/minecraft/block/Material;Lnet/minecraft/block/MapColor;)V
+extendable  method    net/minecraft/block/SaplingBlock              <init>  (Lnet/minecraft/block/sapling/SaplingGenerator;Lnet/minecraft/block/AbstractBlock$Settings;)V
```

Comment:
Since this isn't exposed by the api, we should probably revert it.
```suggestion
```

Reply:
Yeah, `SimpleSaplingBlock` is a temporary solution for the time being.